Control coverage/Feature/DataShare.md

```diff
@@ -0,0 +1,82 @@
+# Data Factory
+**Resource Type:** Microsoft.DataFactory/factories
+
+
+<!-- TOC -->
+
+- [Azure_DataShare_Audit_Enable_Diagnostic_Settings](#azure_datafactory_audit_enable_diagnostic_settings)
+
+<!-- /TOC -->
+<br/>
+
+___ 
+
+## Azure_DataShare_Audit_Enable_Diagnostic_Settings
+ 
+
+### Display Name 
+Enable Security Logging in Azure Data Share
+
+### Rationale 
+Auditing logs and metrics must be enabled as they provide details for investigation in case of a security breach for threats.
+
+### Control Settings 
+```json 
+{
+    "DiagnosticForeverRetentionValue": "0",
+    "DiagnosticMinRetentionPeriod": "90",
+    "DiagnosticLogs": [
+        "Shares",
+        "ShareSubscriptions",
+        "SentShareSnapshots",
+        "ReceivedShareSnapshots"
+    ],
+    "DiagnosticMetrics": [
+        "AllMetrics"
+    ]
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> Diagnostic setting meet the following conditions:
+>   1. Diagnostic logs are enabled.
+>   2. At least one of the below setting configured:
+>       a. Log Analytics.
+>       b. Storage account with min Retention period of 90 or forever(Retention period 0).
+>       c. Event Hub.
+> 
+> **Failed:** 
+> If any of the below conditions are meet:
+>   1. Diagnostic setting meet the following conditions:
+>       a. All diagnostic logs are not enabled.
+>       b. All below settings are not configured:
+>          i. Log Analytics.
+>          ii. Storage account (with min Retention period of 90 or forever(Retention period 0).
+>          iii. Event Hub.
+>   2. Diagnostics setting is disabled for resource.
+
+ 
+### Recommendation 
+
+- **Azure Portal** 
```

Comment:
Steps are different then other resource types?

Reply:
fixed. But we need to update the code.

Reply:
what code?

Reply:
In our AzTS code in extention, the steps i mentioned here were from the recommendation section of the control Json